tasks/get_amq_broker_distribution.yml

```diff
@@ -71,3 +71,11 @@
     - broker
     - ownership
   shell: chown -R "{{ amq_broker_user | default(amq_broker_default_user) }}":"{{ amq_broker_user | default(amq_broker_default_user) }}" {{ amq_broker_install_dest }}
+
+- name: Download JDBC DB connection driver
+  when:
+    - amq_broker_db_driver_url is defined and amq_broker_db_driver_url
+  get_url:
```

Comment:
I think it would be good to be able to turn on/off the certificate validation (via validate_certs option).

Reply:
You mean when downloading packages from url as general for whole role?

Reply:
Implemented in the last commit. Easyfix.